### ISSUE TITLE
Authentication failed on git push

### ISSUE THREAD

**Issue by svenruetz** [iozag/vsts-git-tasks#14]
Hi,

we want to do an empty commit with some build information during a TFS build and I tried to set up the write access using "Enable Git Remote Access" and "Restore Git Remote" from your tools.

I enabled OAuth token support in the build settings. When I try to push my commit to the repository I receive the error 
```
fatal: Authentication failed for 'https://OAuth:********@my-tfs-server:443/myRepo/_git/myRepo/'
```

I tried "git push" as command line task and inside a powershell script, same behaviour.

Any ideas where to find further information on the issue?

Thank you for your help.

We use TFS 2015 (Version 14.102.25423.0)

**Comment by svenruetz** [comment]
Not the right place to ask for help?

**Comment by tayzer** [comment]
Getting the same error, did you ever find a fix?

**Comment by svenruetz** [comment]
No, I abandoned this solution and created a PowerShell script we now use to tag our builds. So no empty commits but annotated tags. We used a .netrc file for autologin to be able to push. I hope this is a helpful hint ;)

**Comment by FreeFrags** [comment]
Any updates? Im getting the same error.

im using a bitbucket repo.

`fatal: Authentication failed for 'https://OAuth:***@bitbucket.org:443/MyCompany/MyRepo.git/'`
